### PR TITLE
Add online package list URLs next to depexts in README.

### PR DESCRIPTION
**Pull request by dbuenzli**

### CODE REVIEW

**Comment by dbuenzli** [comment]
It would be nice to also add a `docker` command to get a shell that corresponds to what the OCaml respository's CI runs. /cc @avsm

**Comment by avsm** [comment]
thanks!

File: README.md [hunk 1]
```diff
@@ -7,24 +7,26 @@ in their definitions).
...
+|------------------------|---------------------------------------------------------|
+| `alpine` `linux`       | https://pkgs.alpinelinux.org                            |
+| `archlinux` `linux`    | https://www.archlinux.org/packages/                     |
+| `centos` `linux`       | http://centos-packages.com/                             |
+| `debian` `linux`       | https://www.debian.org/distrib/packages#search_packages |
+| `dragonfly`            |                                                         |
+| `fedora` `linux`       | https://apps.fedoraproject.org/packages/                |
+| `freebsd`              | https://www.freebsd.org/ports/index.html                |
+| `gentoo`               | https://packages.gentoo.org/                            |
+| `homebrew` `osx`       | http://braumeister.org/                                 |
```
Comment (liyishuai) [inline README.md:20]: This URI is moved permanently to https://sitemap.filecroco.com/n/1.html, which seems irrelevant to Homebrew.
